rasa/core/agent.py

```diff
@@ -782,6 +836,65 @@ def _create_ensemble(
                 "policies, or a policy ensemble".format(passed_type)
             )
 
+    @staticmethod
+    def load_local_model(
+        model_path: Text,
+        interpreter: Optional[NaturalLanguageInterpreter] = None,
+        generator: Union[EndpointConfig, "NLG"] = None,
+        tracker_store: Optional["TrackerStore"] = None,
+        action_endpoint: Optional[EndpointConfig] = None,
+    ) -> "Agent":
+        if os.path.isfile(model_path):
+            model_archive = model_path
+        else:
+            model_archive = get_latest_model(model_path)
+
+        if model_archive is None:
+            logger.warning("Could not load local model in '{}'".format(model_path))
+            return Agent()
+
+        working_directory = tempfile.mkdtemp()
+        unpacked_model = unpack_model(model_archive, working_directory)
+
+        return Agent.load(
+            unpacked_model,
+            interpreter=interpreter,
+            generator=generator,
+            tracker_store=tracker_store,
+            action_endpoint=action_endpoint,
+        )
+
+    @staticmethod
+    def load_from_remote_storage(
+        remote_storage: Text,
+        model_name: Text,
+        interpreter: Optional[NaturalLanguageInterpreter] = None,
+        generator: Union[EndpointConfig, "NLG"] = None,
+        tracker_store: Optional["TrackerStore"] = None,
+        action_endpoint: Optional[EndpointConfig] = None,
+    ) -> "Agent":
+        from rasa.nlu.persistor import get_persistor
+
+        p = get_persistor(remote_storage)
+        if p is not None:
+            target_path = tempfile.mkdtemp()
+            p.retrieve(model_name, target_path)
+
+            return Agent.load(
+                target_path,
+                interpreter=interpreter,
+                generator=generator,
+                tracker_store=tracker_store,
+                action_endpoint=action_endpoint,
+            )
+
+        return Agent(
```

Comment:
`Agent.load` will raise an `ValueError` if `target_path` is None. Calling `Agent()` directly, will not fail if `target_path` is not set.
However, I guess it is better to return `None` in case the persistor does not exists as it indicates a wrong configuration.

Reply:
ah, haven't seen that it's once the load function and once `Agent(...)`